pkg/service/ioservice_sip.go

```diff
@@ -49,8 +49,16 @@ func (s *IOInfoService) matchSIPDispatchRule(ctx context.Context, trunk *livekit
 
 func (s *IOInfoService) EvaluateSIPDispatchRules(ctx context.Context, req *rpc.EvaluateSIPDispatchRulesRequest) (*rpc.EvaluateSIPDispatchRulesResponse, error) {
 	log := logger.GetLogger()
-	log = log.WithValues("toUser", req.CalledNumber, "fromUser", req.CallingNumber)
-	trunk, err := s.matchSIPTrunk(ctx, req.SipTrunkId, req.CallingNumber, req.CalledNumber)
+	log = log.WithValues("toUser", req.CalledNumber, "fromUser", req.CallingNumber, "src", req.SrcAddress)
+	if req.SrcAddress == "" {
```

Comment:
do we want fallthrough if `req.SrcAddress` is empty`?

Reply:
Yes, otherwise it may break existing deployments. I'll add a TODO to later return an error here.